rust/operator-binary/src/spark_k8s_controller.rs

```diff
@@ -155,6 +155,11 @@ pub enum Error {
     CreateVolumes {
         source: stackable_spark_k8s_crd::Error,
     },
+    #[snafu(display("Failed to update status for application [{name}]"))]
```

Comment:
nit(optional): According to our style guide we space those apart when the have macro attributes or documentation on top
```suggestion

    #[snafu(display("Failed to update status for application [{name}]"))]
```